readme.md

```
@@ -110,6 +110,132 @@ const listTxnParams = {
 const txns = await node.listTransactions(listTxnParams);
 ```
 
+#### Galoy GraphQL (Blink, Flash, and compatible deployments)
```

markdownlint-cli2 (0.23.1)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)